src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -225,6 +226,7 @@ def load_arguments(self, _):
             c.argument('kubelet_config', type=str)
             c.argument('linux_os_config', type=str)
             c.argument('enable_encryption_at_host', options_list=['--enable-encryption-at-host'], action='store_true')
+            c.argument('enable_ultra_ssd', options_list=['--enable-ultra-ssd'], action='store_true')
```

Comment:
The newly added parameter "enable_ultra_ssd" will be exposed as an option in hyphenated form (--enable-ultra-ssd) automatically, which is exactly the same as that specified in "options_list". Maybe you can omit this, or change the above  context ("aks create") to make them consistent .

Reply:
fixed.